Part3_InterruptRoutingSetupInAChipset.html

```diff
@@ -30,24 +30,24 @@ <h2>Theory</h2>
 
 	<img src="https://habrastorage.org/webt/mc/ra/hj/mcrahjn-owk9qrcmyiixcmhomcq.png" />
 
-	<p>On these pictures the mapping 'PCI device → PIR' is presented fairly abstract, actually it is a little bit more complicated. In a reality every PCI device has 4 interrupt lanes (INTA#, INTB#, INTC#, INTD#). Also every PCI device can have up to 8 so called 'functions' (which actually trigger interrupts) and each of these functions can connect to the one of the INTx# lane. Which INTx# interrupt will trigger each of the functions either is fixed in the hardware, or is defined by the device software configuration.<p/>
+	<p>On these pictures the mapping 'PCI device → PIR' is fairly abstract - in reality it is a little bit more complicated. In reality every PCI device has four interrupt lanes (INTA#, INTB#, INTC#, INTD#). Also, every PCI device can have up to 8 so called 'functions' (which actually trigger interrupts) and each of these functions can connect to one of the INTx# lanes. Which INTx# interrupt will trigger each of the functions is either fixed in the hardware, or is defined by the device software configuration.<p/>
```

Comment:
This way 'in reality' is repeated twice in a row. If 'actually' can't be used in this case, could we use any different word?

Reply:
Good point, I think "it is actually" would work then.